3-reveal/migrations/5-IRS/4-Zambia-2020/deploy/zambia_irs_district_performance.psql

```diff
@@ -0,0 +1,70 @@
+-- Deploy reveal_irs_zambia_2020:zambia_irs_district_performance to pg
+-- requires: zambia_irs_spray_event
+
+BEGIN;
+
+SET search_path TO :"schema",public;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS zambia_irs_district_performance AS
+SELECT
+  public.uuid_generate_v5(
+    '6ba7b810-9dad-11d1-80b4-00c04fd430c8',
+    concat(district_id, plan_id)
+  ) AS id,
+  district_id,
+  district_name,
+  plan_id,
+  days_worked,
+  found,
+  sprayed,
+  refused,
+  other_reason,
+  to_char(start_time,'HH24:MI') AS start_time,
+  to_char(end_time,'HH24:MI') AS end_time,
+  to_char(end_time -start_time ,'HH24:MI') AS field_duration,
+  COALESCE(found / NULLIF(days_worked, 0)::DECIMAL, found) AS avg_found,
+  COALESCE(sprayed / NULLIF(days_worked, 0)::DECIMAL, sprayed) AS avg_sprayed,
+  COALESCE(refused / NULLIF(days_worked, 0)::DECIMAL, refused) AS avg_refused,
+  bottles_full AS bottles_full,
+  bottles_accounted AS bottles_accounted,
+  bottles_empty AS bottles_empty,
+  bottles_lostdamaged AS bottles_lostdamaged,
+  bottles_start AS bottles_start,
+  daily_found AS daily_found,
+  daily_sprayed AS daily_sprayed,
+  found_diff AS found_diff,
+  sprayed_diff AS sprayed_diff,
+  bottles_empty / daily_sprayed::DECIMAL AS usage_rate,
+  CASE
+    WHEN (found_diff = 0 AND sprayed_diff = 0 ) THEN TRUE ELSE FALSE
+  END AS data_quality_check
+FROM (
+  SELECT 
+    DISTINCT (spray_event.plan_id) as plan_id,
```

Comment:
here again I'm a little confused - does this only ever return one row per plan?

Reply:
I think am wrong on this let me make some changes